pkg/keymanagementprovider/azurekeyvault/types/types.go

```diff
@@ -37,4 +37,6 @@ type KeyVaultValue struct {
 	Name string `json:"name" yaml:"name"`
 	// the version of the Azure Key Vault certificate/key
 	Version string `json:"version" yaml:"version"`
+	// the number of versions to keep in the cache
+	VersionHistoryLimit int `json:"versionHistoryLimit" yaml:"versionHistoryLimit"`
```

Comment:
@akashsinghal @binbin-li what are your thoughts on adding a VersionHistory field here with the `sortVersionHistory ` and `GetSortedVersions` as methods?

Reply:
+1 to binbin's comment